src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1426,5 +1426,13 @@ internal enum ErrorCode
         ERR_TypeInferenceFailedForImplicitlyTypedOutVariable = 8197,
         ERR_ExpressionTreeContainsOutVariable = 8198,
         #endregion diagnostics for out var
+
+        // gap here; available 8149-8499
+
+        #region diagnostics for instrumentation
+
+        ERR_InvalidInstrumentationKind = 8501,
```

Comment:
@drognanar did this. (I'll be adding notes to myself so that I can figure out what remains to be done.)